ferrocene/packages.toml

```diff
@@ -71,27 +71,28 @@ subset = "default"
 
 [groups.cross-compilation]
 targets = [
+  "aarch64-unknown-ferrocene.certified",
+  "aarch64-unknown-ferrocene.facade",
```

Comment:
if you're going to reorder these, could you mark them as "// tidy-start-alphabetical" please?